### ISSUE TITLE
Add Indexed in Cloudflare AMP Cache?

### ISSUE THREAD

**Issue by dknecht** [ampproject/ampbench#11]
Add a section for if pages are properly index in AMP cache and can be accessed.

**Comment by pietergreyling** [comment]
@dknecht Is there documentation of the Cloudflare URL scheme? Something similar to the [AMP Cache URL format](https://developers.google.com/amp/cache/overview#amp-cache-url-format) docs?

**Comment by dknecht** [comment]
Yes.  I put these as place holders.  We are going to fully document all of this shortly.

**Comment by dknecht** [comment]
Cloudflare uses an identical interface to the AMP Project.  With the additional interface mentioned in #12

**Comment by dknecht** [comment]
@pietergreyling  is this something you want us to do pull for?

**Comment by pietergreyling** [comment]
@dknecht Considering the open spirit of AMP my preference would be to not extend the core AMPBench code with 3P checks and validations (such as for 3P AMP Caches etc), but rather to encourage the AMP community to create their own open source code forks within which any validations for 3P-specific extensions can be implemented. Extended "AMPBench-instances" can then be hosted by these AMP Ecosystem 3P service providers which will give their consumers best-of-breed support.
